website/docs/r/org.html.markdown

```diff
@@ -60,6 +60,7 @@ The following arguments are supported:
 * `can_publish_external_catalogs` - (Optional; *v3.6+*) - True if this organization is allowed to publish external catalogs. Default is `false`.
 * `can_subscribe_external_catalogs` - (Optional; *v3.6+*) - True if this organization is allowed to subscribe to external catalogs. Default is `false`.
 * `delay_after_power_on_seconds` - (Optional) - Specifies this organization's default for virtual machine boot delay after power on. Default is `0`.
+* `metadata` - (*v3.6+*) - (Optional) Key value map of metadata to assign to this organization.
```

Comment:
I see almost all of the (old) lines above (except the name and full_name params) have a wrong format for the commentary data. And you have adapted it below too in favor of consistency :)

This is the correct format:

```suggestion
* `metadata` - (Optional; *v3.6+*) Key value map of metadata to assign to this organization.
```

Would you adjust all the above lines too to fit this format (single parenthesis and no hyphen after them)?

Reply:
Good eye! Changed, thanks!